test/entity.js

```diff
@@ -591,7 +591,17 @@ describe('entity', function() {
     });
 
     it('should respect excludeFromIndexes', function() {
-      var value = 'Stephen';
+      var value1 = 'Stephen';
```

Comment:
I love this :)

Reply:
That's the Stephen++ clone army. It keeps on incrementing. :D 
Just found it easier for testing with different values in different places.